schema.sql

```diff
@@ -7,3 +7,14 @@ CREATE TABLE animals (
     weight_kg DECIMAL
     species VARCHAR(255)
 );
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INTEGER
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
```

Comment:
- [ ] Dear @bohaz here in your schema we have a requirement to Remove column `species`, Add column `species_id` which is a foreign key referencing `species` table, and  Add column `owner_id` which is a foreign key referencing the `owners` table. Please add the appropriate queries to do so.